.ci/load/scripts/load_agent.sh

```diff
@@ -86,11 +87,15 @@ function appIsReady() {
 }
 
 function waitForApp() {
+    counter=1
     while :
     do
         if appIsReady; then 
             break 
+        elif [ $counter -gt $POLL_TIMEOUT ]; then
```

Comment:
[minor] maybe echo a small message to explain why we are giving up

Reply:
Good suggestion. Thanks! I have added an explanatory message.